### ISSUE TITLE
[Decision needed] Musicxml import produces MetronomeMark objects for each staff of multistaff part

### ISSUE THREAD

**Issue by jacobtylerwalls** [cuthbertLab/music21#1133]
**music21 version**
dev

**Problem summary**
Importing multistaff parts from musicxml (e.g. keyboard), produces MetronomeMark objects on each `PartStaff`. Great for querying. Exporting, though, will print the marks on both staves.

**Steps to reproduce**

Insert the tag @greg-chapman-dev mentioned in https://github.com/cuthbertLab/music21/pull/1131#issuecomment-921190744 into corpus/demos/two-parts.xml
```
   </direction>
      <direction placement="above">
        <direction-type>
          <metronome parentheses="no">
            <beat-unit>eighth</beat-unit>
            <per-minute>82</per-minute>
      </metronome>
    </direction-type>
    <staff>1</staff>
    <sound tempo="41" />
```
```
>>> s = corpus.parse('demos/two-parts', forceSource=True)
>>> len(s[tempo.MetronomeMark])
2
```

**Expected vs. actual behavior**
A design decision is needed here. Since inception music21 has been prioritizing extraction and querying of piano music over exporting it. I got export working in v.6.5 about a year ago, and now we're growing into the edge cases.

From a totally pedantic stream representation standpoint, there is no MetronomeMark in the second PartStaff, so maybe it shouldn't be there. I just tested if we can still query for it from the stream context. We can, using some of the new tools in v7, but it's involved:

I deleted the second metronome mark:
```python
>>> len(s.parts.last()["Measure"][0]["MetronomeMark"])  # I removed the superfluous tempo mark
0
>>> s.parts.last()["Measure"][0].getContextByClass('MetronomeMark') is None
True
>>> s.parts.last()["Measure"][0].getContextByClass('MetronomeMark', getElementMethod='all') 
<music21.tempo.MetronomeMark andantino Eighth=82.0>
```

So we need to decide on the stream representation.

**Comment by jacobtylerwalls** [comment]
Maybe there could be a .tempo lookup for all streams that is overridden for PartStaff to run the context search by default. Or we leave the duplicates in and just hideObjectOnPrint on the second one.

**Comment by gregchapman-dev** [comment]
I would prefer any solution that represents pedantically what is actually in the original score (i.e. "Don't lie").  Making searching simpler by auto-adding context for PartStaffs sounds like a good thing as well.

**Comment by jacobtylerwalls** [comment]
Oops, we have tooling for this already. `.numberImplicit`, see #710 where I did this for MIDI import. Let's do the same for MusicXML import for consistency.

See also discussion at https://musescore.org/en/node/137091

**Comment by jacobtylerwalls** [comment]
Edited above to clarify this is a regression in 0159785.

**Comment by mscuthbert** [comment]
yes, this should be hidden.  I can't remember whether .numberImplicit or .style.hideObjectOnPrint is better in this case.
